onnx/backend/test/case/node/bitwiseand.py

```diff
@@ -18,14 +18,14 @@ def export() -> None:
         )
 
         # 2d
-        x = np.random.randn(3, 4).astype(np.int32)
-        y = np.random.randn(3, 4).astype(np.int32)
+        x = np.random.randint(10, size=(3, 4)).astype(np.int32)
+        y = np.random.randint(10, size=(3, 4)).astype(np.int32)
```

Comment:
Could we cover the negative part of signed integer types, e.g. `np.random.randint(-10, 10, size(3, 4)).astype(np.int32)` so that the tests are more reliable

Reply:
Sounds good. Updated. PTAL. Thanks!